src/pages/waiting-room-to-car/cat-be/components/vehicle-checks/vehicle-checks.ts

```diff
@@ -53,4 +54,27 @@ export class VehicleChecksCatBEComponent {
   hasDrivingFault(): boolean {
     return this.vehicleChecksScore.drivingFaults > 0;
   }
+
+  invalidVehicleChecks(c: FormControl): { vehicleChecks: boolean } {
+    return { vehicleChecks: false };
+  }
+
+  validateVehicleChecks() {
+    if (!this.formControl) {
+      this.formControl = new FormControl({
+        value: 'Select questions',
+        disabled: false,
+      },
+      [this.everyQuestionHasOutcome() ? null : this.invalidVehicleChecks]);
```

Comment:
Think should create a custom validator  with the check and return and add that to the form control.

Reply:
look at pass-certificate-number.ts (validatePassCertificate method) as an example custom validator.

Reply:
Thanks, I will take a look.